### ISSUE TITLE
Session recording

### ISSUE THREAD

**Issue by HRNasr** [casdoor/awesome-auth#34]
In this project, is there a plan to record the meetings and paly them online/offline for the managers of that organization (defined organization)?

**Comment by hsluoyz** [comment]
@HRNasr what do you mean by meetings? Casvisor is a remote desktop management platform, not a video meeting platform like Zoom.

**Comment by HRNasr** [comment]
No. The aim is to record the sessions established on the said protocols by the guac module. For future review

**Comment by HRNasr** [comment]
Or having the ability to see live user sessions by the system administrator and even share sessions with her for mutual understanding

**Comment by hsluoyz** [comment]
@HRNasr can you please avoid using the term "meeting", does it mean remote desktop like RDP or VNC session?

**Comment by HRNasr** [comment]
Yes.  mean rdp, ssh, vnc sessions

**Comment by HRNasr** [comment]
or session join
session sharing
for connect other admin and help
